cds_migrator_kit/videos/weblecture_migration/transform/xml_processing/rules/video_lecture.py

```diff
@@ -82,5 +109,124 @@ def event_speakers(self, key, value):
     """Translates event_speakers."""
     name = value.get("p").strip()
     contributor = {"name": name, "role": "Speaker"}
-    # TODO contributor affiliation will be implemented
+    affiliation = value.get("u", "")
+    if affiliation:
+        contributor.update({"affiliations": [affiliation]})
     return contributor
+
+
+@model.over("url_files", "^8564_")
+@for_each_value
+@require(["u"])
+def url_files(self, key, value):
+    """Detects 8564 files."""
+    url = value.get("u")
+    if "digital-memory" in url:
+        return {
+            "digitized": {
```

Comment:
Be aware that there was some work from a summer student to introduce a `_digitization` field (see [pr](https://github.com/CERNDocumentServer/cds-videos/pull/1932)) maybe we can align on the key name and subfields